lms/djangoapps/mobile_api/social_facebook/groups/tests.py

```diff
@@ -0,0 +1,171 @@
+"""
+Tests for groups
+"""
+
+from django.core.urlresolvers import reverse
+from rest_framework.test import APITestCase
+from xmodule.modulestore.tests.django_utils import ModuleStoreTestCase
+from courseware.tests.factories import UserFactory
+
+
+class TestGroups(ModuleStoreTestCase, APITestCase):
```

Comment:
These tests need to use the `httpretty` decorators.  We cannot rely upon a valid connection to Facebook to run our unit tests.  Also, we don't want to create/delete/edit actual groups on the Facebook server during test runs.